### ISSUE TITLE
[RFC] Adding columns at Runtime (EAV principle)

### ISSUE THREAD

**Issue by DaHaiz** [propelorm/Propel2#44]
Hey Guys,

i'm currently have to face with a product database. As everyone knows, different kinds of products can have different attributes and also a different count of attriutes. for example a TV has screenwidth(inch=>integer), or hdmi (1/0=>smallint) and a tshirt has size(X/M/L=>varchar) etc. At buildtime it is not possible to decided which fields an which type of products will be used later.
the magento shop system solves this problem with the entity-attribute-value princple. For me as i am using propel thats kind of a problem. For sure it would be possible to add the attributes as a new Entity and therefore a new table. I would then have to add magic to __call(), but i think this problem is sth, that would be useful for very many propel users.
In Short: It would be great if you could make propel even greater and implement it directly into the core behaviors as this is a bigger feature i think and it should be implemented with all the background knowledge of propel which is possible to gain the best out of this feature. Discussion opened :)

greets,
DaHaiz
